src/main/java/Storage.java

```diff
@@ -0,0 +1,57 @@
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+
+/** This class handles the saving and loading of tasks. */
+public class Storage {
+    private static String path;
+
+    /**
+     * Sets the path to load and save from.
+     *
+     * @param path The path to load and save from.
+     */
+    public static void setPath(String path) {
+        Storage.path = path;
+    }
+
+    /**
+     * Save tasks.
+     *
+     * @param storage The list of tasks to store.
+     */
+    public static void saveTasks(ArrayList<Task> storage) {
+        try {
+            FileOutputStream fs = new FileOutputStream(path);
+            ObjectOutputStream oos = new ObjectOutputStream(fs);
+            oos.writeObject(storage);
+            oos.close();
```

Comment:
May want to consider closing it only after program is close. Flushing it at each instance/operation instead